db/schema.rb

```diff
@@ -0,0 +1,18 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
There is no `fields` table in this file. Did you run `rake db:migrate`?

Reply:
Yes, ran it, but after it saw some pitfalls in migration file and made rollback. After correcting migration file I didn't run it again. As I understand, together with migration file I have to upload updated schema.rb ?

Reply:
Correct